frontend/packages/kubevirt-plugin/src/components/create-vm/dev-console/use-catalog-vm-templates.tsx

```diff
@@ -55,7 +55,8 @@ const normalizeVmTemplates = (
     const sourceProvider = !isTemplateSourceError(sourceStatus) && sourceStatus?.provider;
     const imgUrl = getTemplateOSIcon(tmp);
     const workloadType = getWorkloadProfile(tmp) || t('kubevirt-plugin~Not available');
-    const flavor = getTemplateFlavorDesc(tmp, false);
+    const flavorData = getTemplateFlavorData(tmp);
+    const flavor = t('kubevirt-plugin~{{flavor}}: {{cpu}} CPU | {{memory}}', flavorData);
```

Comment:
this is getting translated again in line 78

Reply:
Done!